orange_silver_2.rb

```diff
@@ -0,0 +1,72 @@
+#defining our Orange Tree Class
+class OrangeTree
+  #assigning read/write abilities to all of the attributes
+  attr_reader :age, :is_dead, :max_age, :height, :count
+
+  def initialize
+    @age = 0
+    @height = 0
+    @count = 0
```

Comment:
I wasn't sure what role `count` was going to play for a minute. You might want to pick a name a little more descriptive of the data the variable will represent, maybe something like `orange_count`?